stories/components/toolbar.stories.tsx

```diff
@@ -21,6 +21,9 @@ export default {
     vertical: false,
     intent: 'none',
     disabled: false,
+    minimal: false,
```

Comment:
Can you make this true since it used to be the default and we actually mostly want to use the minimal version.